style.css

```diff
@@ -62,3 +62,177 @@
   position: absolute;
   padding-top: 30px;
 }
+
+.works {
+  display: grid;
+  row-gap: 20px;
+  grid-template-rows: auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  margin-left: 28px;
+  width: 327px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
```

Comment:
Kindly note that the margins in your page are different from those in the figma design. I suggest you fix this
- [ ] Your design
<img width="521" alt="Screenshot 2022-09-24 at 10 38 58" src="https://user-images.githubusercontent.com/48546992/192086524-18451712-52fc-4001-b02b-926427f4dbb7.png">


- [ ] Figma design
<img width="199" alt="Screenshot 2022-09-24 at 10 37 33" src="https://user-images.githubusercontent.com/48546992/192086561-30c55f5a-7d55-4a49-96d7-6cdb7cf7b999.png">